### ISSUE TITLE
Props works wrong when component is written in index.tsx

### ISSUE THREAD

**Issue by Tarocch1** [doczjs/docz#1224]
## Bug Report

**Describe the bug**

Props works wrong when component is written in `index.tsx`.

**To Reproduce**

1. Follow the steps in https://github.com/doczjs/docz/tree/master/examples/typescript#download-manually, use docz typescript example.

2. npm run dev. Everything is normal.

![image](https://user-images.githubusercontent.com/15124068/68075339-565e4b80-fde1-11e9-8a39-5d070df296ec.png)

3. Rename `src\components\Button.tsx` to `src\components\index.tsx`

4. Edit `src\components\Button.mdx`:

```mdx
import { Button } from './index.tsx';

<Props of={Button} />
```

5. npm run dev. Props works wrong.

![image](https://user-images.githubusercontent.com/15124068/68075295-b6082700-fde0-11e9-840c-b0f1614503bc.png)

6. Undo rename and edit.

7. npm run dev. Everything is normal again.

**Expected behavior**

Props works fine when component is written in `index.tsx`.

**Environment**

- docz version: 2.0.0-rc.68
- OS: Windows 10
- Node/npm version: Node 12/npm 6

**Comment by rakannimer** [comment]
Hey @Tarocch1 

Thanks for reporting this issue ! It will hopefully be fixed in the next docz release candidate.

Meanwhile, you can fix it *temporarily* from your end.

In your `doczrc.js` file add a `filterComponents` methods : 

```js
// doczrc.js
export default {
  typescript: true,
  filterComponents: files =>
    files.filter(
      filepath =>
        /\/[A-Z]\w*(\/index)?\.(js|jsx|ts|tsx)$/.test(filepath) ||
        filepath.includes("index")
    )
};
```

and it should work as expected.

I'll update this issue once the issue is resolved from `docz`'s side 👍

**Comment by rakannimer** [comment]
Fixed in `2.0.0-rc.69`. Please feel free to re-open if it still happens.
